ingestion-lambda/src/handler.ts

```diff
@@ -124,10 +125,12 @@ export const main = async (event: SQSEvent): Promise<SQSBatchResponse> => {
 
 						const snsMessageContent = safeBodyParse(body);
 
+						const supplier = lookupSupplier(snsMessageContent['source-feed']!)
```

Comment:
using the `!` assertion here feels a bit odd to me; I think I'd prefer to either explicitly check that `snsMessageContent['source-feed']` is defined before invoking `lookupSupplier`, or else have `lookupSupplier` explicitly handle `undefined` as an input..

not blocking but I'd be interested to hear what you think

Reply:
Agreed, I might have left it while testing it locally.